canvas_manage_course/requirements/base.txt

```diff
@@ -1,17 +1,22 @@
 boto3==1.4.4
-Django==1.9.12
-cx-Oracle==5.2.1
+Django==1.11.9
+cx-Oracle==6.0.3
 django-cached-authentication-middleware==0.2.1
 django-redis-cache==1.7.1
 hiredis==0.2.0
 kitchen==1.2.4
 ndg-httpsclient==0.4.2
-redis==2.10.5
-psycopg2==2.6.2
+redis==2.10.6
+psycopg2==2.7.3.2
 requests==2.13.0
 
-git+ssh://git@github.com/penzance/canvas_python_sdk.git@v0.8.4#egg=canvas-python-sdk==0.8.4
-git+ssh://git@github.com/Harvard-University-iCommons/django-auth-lti.git@v1.2.5#egg=django-auth-lti==1.2.5
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.18.2#egg=django-icommons-common[async]==1.18.2
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-ui.git@v1.3.1#egg=django-icommons-ui==1.3.1
-git+ssh://git@github.com/Harvard-University-iCommons/django-canvas-lti-school-permissions@v0.5.2#egg=django-canvas-lti-school-permissions==0.5.2
+git+ssh://git@github.com/penzance/canvas_python_sdk.git@v0.10.2#egg=canvas-python-sdk==0.10.2
+git+ssh://git@github.com/Harvard-University-iCommons/django-auth-lti.git@v1.2.9#egg=django-auth-lti==1.2.9
+
+# NOTE: Including the django-pgjson library here. This is required by 'async' within django-icommons-common, but
+# there is fix for a bug for django<=1.10 , thgat is still in a commit but is not in the released version. Unable to specify
+# a commit in the django-icommons-common's setup.py, hence including it in the consuming app till we used switch from pjson to standard json
+git+ssh://git@github.com/djangonauts/django-pgjson.git@30463d210a42b2de#egg=django-pgjson
```

Comment:
This version of pgjson uses import_by_path which is deprecated in Django 1.10+. Causes the deploy to fail.
https://github.com/djangonauts/django-pgjson/commit/30463d210a42b2de#diff-09ac573aadd5de6bf33fc0a97964399a
https://github.com/djangonauts/django-pgjson/issues/45

Reply:
@Chris-Thornton-Harvard :  So the latest Release (tagged 0.31) of django-pgjson from July 2015 ((https://github.com/djangonauts/django-pgjson/releases/tag/0.3.1) uses  the deprecated import_by_path. The second link you have is the reported issue, but is still open. 

I found the link to  this particular commit   - which  a workaround to this issue(Details in the last comment here https://github.com/djangonauts/django-pgjson/issues/43 ), it has been merged to master but not ben tagged.  It worked locally for me, will check why it's failing  while we deploy to dev. Thanks for catching it.